test/framework/twilio-video-no-framework/app.js

```diff
@@ -19,7 +19,7 @@ function getQueryParameters(location) {
 
 const token = (getQueryParameters(location).get('token') || [])[0] || '';
 
-new Twilio.Video.Client().connect({ token: token }).then(room => {
+new Twilio.Video.connect({ token: token }).then(room => {
```

Comment:
@markandrus This might conflict with the same change that I made in `client:media-api-simplification`.